packages/ra-language-french/src/index.ts

```diff
@@ -90,6 +90,8 @@ const frenchMessages: TranslationMessages = {
             not_found:
                 "L'URL saisie est incorrecte, ou vous avez suivi un mauvais lien.",
             yes: 'Oui',
+            unsaved_changes:
+                "Certains changements n'ont pas été enregistrés. Etes-vous sûr(e) de vouloir quitter cette page ?",
```

Comment:
```suggestion
                "Certains changements n'ont pas été enregistrés. Êtes-vous sûr·e de vouloir quitter cette page ?",
```

Reply:
we have "sûr(e)" already in other translations, let's keep parentheses for this one.